bundle/compliance/main.rego

```diff
@@ -3,24 +3,25 @@ package main
 import data.compliance
 import data.compliance.lib.common
 
-# input is a resource
-# data is policy/configuration
+# input contains the resource and the configuration
 # output is findings
 
 resource = input.resource
 
+activated_rules = input.activated_rules
```

Comment:
This will cause the bundle to return the activated_rules upon the result of the decision.
```
{
  "result": [
    {
      "expressions": [
        {
          "value": {
            "activated_rules": "JENIA",
            "findings": [
            ...
            ],
            "metadata": {
              "opa_version": "0.42.0-dev"
            },
            "resource": {
              "group": "root",
              "mode": "644",
              "name": "10-kubeadm.conf",
              "owner": "root",
              "path": "/hostfs/etc/kubernetes/manifests/kube-apiserver.yaml"
            }
          },
          "text": "data.main",
          "location": {
            "row": 1,
            "col": 1
          }
        }
      ]
    }
  ]
}
```
Notice that we have the "activated_rules" field as part of the result.
I'm not sure if this is something that we want.